units/meson.build

```diff
@@ -82,6 +82,8 @@ units = [
         ['sound.target',                        ''],
         ['suspend.target',                      ''],
         ['swap.target',                         ''],
+        ['sys-firmware-efi-efivars.mount',      'not ENABLE_EFI',
+         'sysinit.target.wants/'],
```

Comment:
Yeah, this seems the wrong approach. *If* we go down the route of making the efivars mounts optional, then this .mount unit should be treated the same as other optional units: installed when `ENABLE_EFI` *is true*`, and with its own preset, and enabled/disabled by symlinks after installation.

Reply:
if we can drop that line, then this negated condition can go (just install the unit always): https://github.com/systemd/systemd/blob/e9b88a6d4ec7818a334ee12f50f23706bcc3c7e0/src/shared/mount-setup.c#L108

Otherwise systemd will ignore mount units for this path